content/learn/case-studies/transunion.md

```diff
@@ -0,0 +1,50 @@
+---
+$order: 26
+$title: TransUnion
+$date: Mar 12, 2018
+$category: ecommerce
+$parent: /content/learn/case-studies.html
+class: case-study post-detail
+
+components:
+  - social-share
+
+headline: TransUnion scores more conversions with faster page loads via AMP
+results:
+    - stat: 3%
+      description: increase in conversion
+    - stat: 26% 
```

Comment:
Did you purposely omit the other stat? In the PDF - `1.6s AMP load vs 7.1s non-AMP load`

Reply:
Yes, it looked too long for the web format and it's covered in the body copy.